mern-ecommerce/routes/auth.js

```diff
@@ -9,8 +9,8 @@ const {
 } = require('../controllers/auth');
 const { userSignupValidator } = require('../validator');
 
-router.post('/signup', userSignupValidator, signup);
-router.post('/signin', signin);
-router.get('/signout', signout);
+router.post('/signup', userSignupValidator, signupss);
```

Comment:
The function `signupss` is not defined in the provided context. It seems like it should be `signup` instead of `signupss`.